tests/components/abode/fixtures/devices.json

```diff
@@ -622,7 +622,7 @@
             }
         ],
         "status_icons": [],
-        "icon": "assets/icons/streaming-camaera-new.svg",
+        "icon": "assets/icons/streaming-camera-new.svg",
```

Comment:
These are testing fixtures, did you adjust the tests as well?

Reply:
The tests do not appear to do anything at all to icons. I did look to see if there was a file with a similar name. I couldn't find one. I looked because if there was one, I would have included a rename in the commit.

The data appears to relate to https://github.com/MisterWil/abodepy/issues/60

In principle, a fixture should be free to provide any value it likes as opposed to the official value provided by a live device so long as the value it provides is internally consistent for its purpose as a fixture. Since there is no instance of an actual file in the fixture, I believe this change is technically valid. That said, I'm quite willing to close this PR (or any other to which I receive pushback).

Beyond the above, I've sent feedback to goabode suggesting they fix their typo. In general, when people suggest an upstream, I'm happy to send feedback/patches there as well.